nservicebus/azure-storage-persistence/configuration.md

```diff
@@ -61,7 +37,7 @@ When using XML-based configuration, the following properties can be set through
 
 #### Subscription Configuration
 
-The following settings are available for changing the behavior of subscription persistence through the `AzureSubscriptionStorageConfig` section:
+The following settings are available for changing the behavior of subscription persistence:
 
  * `ConnectionString`: Sets the connection string for the storage account to be used for storing subscription information.
   * NServiceBus.Azure Versions 6 and below defaults to `UseDevelopmentStorage=true`.
```

Comment:
Is Version 6 the latest one?. If yes can we remove the version information and just say that default value is `UseDevelopmentStorage=true`?